CODE/logicytics/FileManagement.py

```
@@ -0,0 +1,234 @@
+import hashlib
+import os.path
+import shutil
+import subprocess
+import zipfile
+from datetime import datetime
+from pathlib import Path
+
+
+class FileManagement:
+    @staticmethod
+    def open_file(file: str, use_full_path: bool = False) -> str | None:
```

Ruff (0.8.0)

Boolean-typed positional argument in function definition

(FBT001)